docs/issue_triage_guide.md

```diff
@@ -0,0 +1,39 @@
+# Issue Triage Guide
+
+This guide describes how to triage git issues on the [issue queue](https://github.com/openservicemesh/osm/issues) for Open Service Mesh (OSM).
+
+## Steps for Traige
+1. Identify the cause of the issue as explained below
+2. Tag the issue with relevant tags
+3. Assign the issue
+4. Prioritize the issue with a P1, P2 or P3 label where:
+    - P1: product cannot ship. Should be addressed as soon as possible. Eg: high priority bug
+    - P2: Product cannot ship, but it does not need to be addressed immediately. Eg: high priority feature, lower priority bug
+    - P3: Resolution of the work item is optional based on resources, time, and risk. Eg: a nice to have feature
+5. Size the issue and assign it to a milestone:
+    [WIP]
+6. If you don't have time to resolve it, put links to the code to where you see the problem arising
+7. Assign the issue to someone
+
+
+## Identifying the cause of the issue
+- If it's a bug: 
+    - Reproduce the issue to make sure it is an actual issue
+    - Tag it with the `bug` label
+    - If possible, fix it or else reference links to the code 
+
+- If it's a question: 
+    - Ask around, go through docs, etc.
+    - Respond to the question
+    - Label appropriately
```

Comment:
We do have a "question" label that is probably worth mentioning specifically here.